drivers/builtin/include/mbedtls/asn1.h

```diff
@@ -287,6 +287,28 @@ int mbedtls_asn1_get_int(unsigned char **p,
                          const unsigned char *end,
                          int *val);
 
+/**
+ * \brief               Parse an Integer and return a (pointer, length) pair that
+ *                      refer to its big-endian value as an array of bytes.
+ * \param[in,out] p     On entry, \c *p points to the start of the ASN.1 element.
+ *                      On successful completion, \c *p points to the first byte
+ *                      beyond the ASN.1 element.
+ *                      On error, the value of \c *p is undefined.
```

Comment:
Minor:
```suggestion
 *                      On error, the value of \c *p is unspecified.
```

(In C, it's customary to use “unspecified” for a value and “undefined” for behavior.)

Reply:
Changed in a new commit